Cars_Collision/cars_collision.cpp

```diff
@@ -0,0 +1,234 @@
+#include <iostream>
+#include <cmath>
+#include <vector>
+
+class Object 
+{
+    public:
+    double x_value, y_value;    
+    double velocity,angle;
+
+    Object()
+    {
+        x_value  = 0;
+        y_value  = 0;
+        velocity = 0;
+        angle    = 0;
+    }
+
+    Object(double x_value, double y_value, double velocity, double angle)
+    {
+        this -> x_value  = x_value;
+        this -> y_value  = y_value;
+        this -> velocity = velocity;
+        this -> angle    = angle;  
+    }
+    
+};
+
+void calculateCollisionFunction(const Object& obj1, const Object& obj2, double& collisionTime)
```

Comment:
Following things are wrong with the following function
1) We already know it is a function so no need to add Function in the name
2) For passing POD(Plain Old Data Types) like double int float we should not pass it by reference as it is more in efficient

Reply:
This function can be part of Car class. So than you won't have to expose all variables